mbed_tools/_internal/config_cli.py

```diff
@@ -0,0 +1,46 @@
+"""Exposes a click command which prints information about configuration variables of all child mbed packages."""
+import click
+import pdoc
+from typing import (
+    Iterable,
+    List,
+    cast,
+)
+
+from mbed_devices.mbed_tools import config_variables as mbed_devices_config_variables
+
+
+@click.command()
+def cli() -> None:
+    """Prints information about configuration variables of all child mbed packages."""
+    click.echo(_build_output(_gather_configuration_variables()))
+
+
+def _gather_configuration_variables() -> List[pdoc.Variable]:
```

Comment:
Any reason we don't just pass `mbed_devices_config_variables` directly to `_build_output`? I don't think this single line function buys us much when we only have a single caller?

Reply:
You're right, that's silly.

Reply:
https://github.com/ARMmbed/mbed-tools/pull/7/commits/828bd2539eb02dfd4dcdda15fe7cc79c2bddb7aa